EIPS/eip-5023.md

```diff
@@ -0,0 +1,145 @@
+---
+eip: 5023
+title: Shareable non-transferable non-fungible token standard
+description: A standard interface for shareable non-transferable NFTs
+author: Jarno Marttila (@yaruno), Martin Moravek (@mmartinmo)
+discussions-to: https://ethereum-magicians.org/t/new-nft-concept-shareable-nfts/8681
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-01-28
+requires: 165
+---
+
+## Abstract
+
+The following defines a standard interface for shareable tokens used in [ATARCA project](https://atarca.eu/) (Grant agreement 964678) to create shareable non-transferable non-fungible tokens. The standard allows creation of tokens that can hold value, but cannot be exchanged for rival goods such as money, cryptocurrencies or other goods that hold monetary value. Shareability of tokens can be understood as re-minting of a token for a new recipient e.g. by making a copy of the token and by giving it away while retaining the original one. The sharing and its associated events allow construction of a graph describing who has shared what to which party.
```

Comment:
No external links, and the abstract should be terse.  Also in general it is frowned on to reference specific products/platforms in a standard, as it should be agnostic to any specific application.
```suggestion
This standard allows creation of tokens that can hold value, but cannot be exchanged for rival goods such as money, cryptocurrencies or other goods that hold monetary value. Shareability of tokens can be understood as re-minting of a token for a new recipient e.g. by making a copy of the token and by giving it away while retaining the original one. The sharing and its associated events allow construction of a graph describing who has shared what to which party.
```